src/clarity/forms/_forms.clarity.scss

```diff
@@ -258,34 +270,39 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
                 padding-left: 0;
                 margin-bottom: $clr_baselineRem_1;
 
-                & > label:first-of-type {
-                    position: relative;
+                & > label:first-child,
+                & > label:not(:first-child),
```

Comment:
@Shijir, @mathisscott: Nope. Since we have different flex properties for them on larger screens we have to override them both on small screens. Using only `& > label` has less specificity than mentioning the `first-child` rule and the previous styles override the behavior.